airbyte-integrations/connectors/source-sftp/src/main/java/io/airbyte/integrations/source/sftp/SftpSource.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.source.sftp;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Lists;
+import io.airbyte.commons.util.AutoCloseableIterator;
+import io.airbyte.commons.util.AutoCloseableIterators;
+import io.airbyte.integrations.BaseConnector;
+import io.airbyte.integrations.base.IntegrationRunner;
+import io.airbyte.integrations.base.Source;
+import io.airbyte.protocol.models.*;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SftpSource extends BaseConnector implements Source {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SftpSource.class);
+
+    public static void main(String[] args) throws Exception {
+        new IntegrationRunner(new SftpSource()).run(args);
+    }
+
+    @Override
+    public AirbyteConnectionStatus check(JsonNode config) throws Exception {
```

Comment:
Please add a comment describing what the `check` method implementation does

Reply:
Same here